codegen_glibmm/templates/proxy.cpp.templ

```diff
@@ -102,14 +102,21 @@ std::tuple<{{ method.out_args|join(', ', attribute='cpptype_out') }}>
 {% endfor %}
 {% for prop in interface.properties %}
 {% if prop.readable %}
-{{ prop.cpptype_out }} {{ class_name_with_namespace }}::{{ prop.name }}_get()
+{{ prop.cpptype_out }} {{ class_name_with_namespace }}::{{ prop.name }}_get(bool *ok)
 {
     std::vector<Glib::ustring> props = m_proxy->get_cached_property_names();
     Glib::Variant<{{ prop.variant_type }}> b;
     if (std::find(props.begin(), props.end(), "{{ prop.name }}") != props.end()) {
         m_proxy->get_cached_property(b, "{{ prop.name }}");
+        if (ok) {
+            *ok = true;
+        }
     } else {
-        g_print("Todo: lookup value\n");
+        if (ok) {
+            *ok = false;
+        } else {
+            g_warning("Unhandled error while getting property {{ prop.name }}\n");
```

Comment:
Remove the `\n` (it was there before, too, but it's not needed)

Reply:
Oh you're right, didn't notice it. Fixed!